lib/ohai/plugins/linux/virtualization.rb

```diff
@@ -222,10 +222,15 @@ def nova_exists?
       Ohai::Log.debug("Plugin Virtualization: /dev/lxd/sock exists. Detecting as lxd guest")
       virtualization[:system] = "lxd"
       virtualization[:role] = "guest"
-    elsif File.exist?("/var/lib/lxd/devlxd")
-      Ohai::Log.debug("Plugin Virtualization: /var/lib/lxd/devlxd exists. Detecting as lxd host")
-      virtualization[:system] = "lxd"
-      virtualization[:role] = "host"
+    else
+      ["/var/lib/lxd/devlxd", "/var/snap/lxd/common/lxd/devlxd"].each do |devlxd|
```

Comment:
Can you put a brief note about what versions we're looking for in each of these locations. It'll help years from now when someone reads this and wonders if we still need to check both locations

Reply:
incoming...

it's more about 'how it's installed' as both methods will remain in use for the foreseeable future.  But yea breaking it up by version(s) will lead to a more concise explanation I believe.